tests/functional/pv/pv_services/test_rwop_pvc.py

```diff
@@ -53,41 +53,69 @@ def test_pod_with_same_priority(self, pod_factory, interface):
 
         """
         # Creating a pod
-        self.pod_obj = pod_factory(
+        pod_obj1 = pod_factory(
             pvc=self.pvc_obj,
         )
 
         # Verify that PVCs are reusable by creating new pods
         log.info(f"PVC obj {self.pvc_obj}")
-        self.create_pod_and_validate_pending(pod_factory, interface)
+        pod_obj2 = self.create_pod_and_validate_pending(pod_factory, interface)
```

Comment:
To better distinguish between RWO and RWOP, please create the seocnd pod in the same node where the first pod is running.

Reply:
Done @jilju